src/lib/bootloader/proposal_client.rb

```diff
@@ -153,6 +153,7 @@ def construct_proposal_map
 
       ret["links"] = PROPOSAL_LINKS # use always possible links even if it maybe not used
       ret["raw_proposal"] = Yast::Bootloader.Summary
+      ret["label_proposal"] = Yast::Bootloader.Summary
```

Comment:
so no check for simple_mode param? BTW to make it more efficient you can use for benefit knowledge that never both values are modified, so it can be
```
ret["raw_proposal"] = Yast::Bootloader.Summary
ret["label_proposal"] = ret["raw_proposal"]
```